### PR TITLE
Update bulk-create-publish-labels-using-powershell.md

### PR DESCRIPTION
**Pull request by Sherylt**
Updated to reflect 'retention labels' and 'retention label policies' terminology to align with distinguishing them from 'retention policies' and 'sensitivity labels'.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SecurityCompliance/bulk-create-publish-labels-using-powershell.md [hunk 1]
```diff
@@ -12,32 +12,32 @@ localization_priority: Priority
...
+description: "In Office 365, you can use retention labels to implement a retention schedule for your organization. As a record manager or compliance officer, you might have hundreds of retention labels to create and publish. You can do this through the UI in the Security and Compliance Center, but creating retention labels one at a time is time-consuming and inefficient. By using the script and .csv files provided below, you can bulk create and publish retention labels and retention label policies. First you create a list of the retention labels and a list of the retention label policies in Excel, and then you bulk create the retention labels and retention label policies in those lists by using PowerShell. This makes it easier to create and publish at one time, all of the retention labels that your retention schedule requires."
```
Comment (JohanFreelancer9) [inline SecurityCompliance/bulk-create-publish-labels-using-powershell.md]:
```suggestion
description: "In Office 365, you can use retention labels to implement a retention schedule for your organization. As a record manager or compliance officer, you might have hundreds of retention labels to create and publish. You can do this through the UI in the Security and Compliance Center, but creating retention labels one at a time is time-consuming and inefficient. By using the script and .csv files provided below, you can bulk create and publish retention labels and retention label policies. First you create a list of the retention labels and a list of the retention label policies in Excel, and then you bulk create the retention labels and retention label policies in those lists by using PowerShell. This makes it easier to create and publish all of the retention labels that your retention schedule requires at the same time."
```

File: SecurityCompliance/bulk-create-publish-labels-using-powershell.md [hunk 2]
```diff
@@ -12,32 +12,32 @@ localization_priority: Priority
...
   
-By using the script and .csv files provided below, you can bulk create and publish labels and label policies. First you create a list of the labels and a list of the label policies in Excel, and then you bulk create the labels and label policies in those lists by using PowerShell. This makes it easier to create and publish at one time all of the labels that your retention schedule requires.
+By using the script and .csv files provided below, you can bulk create retention labels and publish retention label policies. First you create a list of the retention labels and a list of the retention label policies in Excel, and then you bulk create the retention labels and retention label policies in those lists by using PowerShell. This makes it easier to create and publish at one time, all of the retention labels that your retention schedule requires.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/bulk-create-publish-labels-using-powershell.md]:
```suggestion
By using the script and .csv files provided below, you can bulk create retention labels and publish retention label policies. First you create a list of the retention labels and a list of the retention label policies in Excel, and then you bulk create the retention labels and retention label policies in those lists by using PowerShell. This makes it easier to create and publish all of the retention labels that your retention schedule requires at the same time.
```

File: SecurityCompliance/bulk-create-publish-labels-using-powershell.md [hunk 3]
```diff
@@ -12,32 +12,32 @@ localization_priority: Priority
...
   
-## Step 1: Create a .csv file for creating the labels
+## Step 1: Create a .csv file for creating the retention labels
 
-First you create a .csv file that contains a list of your labels with their retention settings. You can use the sample below as a template by copying it into Excel, converting the text to columns (in Excel \> **Data** tab \> **Text to Columns** \> **Delimited** \> **Comma** \> **General**), and then saving the worksheet as a .csv file in a location that's easy to find.
+First you create a .csv file that contains a list of your retention labels with their settings. You can use the sample below as a template by copying it into Excel, converting the text to columns (in Excel \> **Data** tab \> **Text to Columns** \> **Delimited** \> **Comma** \> **General**), and then saving the worksheet as a .csv file in a location that's easy to find.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/bulk-create-publish-labels-using-powershell.md]:
```suggestion
First create a .csv file that contains a list of your retention labels with their settings. You can use the sample below as a template by copying it into Excel, converting the text to columns (in Excel \> **Data** tab \> **Text to Columns** \> **Delimited** \> **Comma** \> **General**), and then saving the worksheet as a .csv file in a location that's easy to find.
```

File: SecurityCompliance/bulk-create-publish-labels-using-powershell.md [hunk 4]
```diff
@@ -12,32 +12,32 @@ localization_priority: Priority
...
   
 For more information about the parameter values for this cmdlet, see [New-ComplianceTag](https://go.microsoft.com/fwlink/?linkid=866511).
   
 Notes:
   
-- If you don't provide a source file for creating labels, the script moves on and prompts you for the source file for publishing labels (see the next section), and the script will publish only existing labels.
+- If you don't provide a source file for creating retention labels, the script moves on and prompts you for the source file for publishing retention labels (see the next section), and the script will publish only existing retention labels.
     
-- If the .csv file contains a label with the same name as one that already exists, the script skips creating that label. No duplicate labels are created.
+- If the .csv file contains a retention label with the same name as one that already exists, the script skips creating that retention label. No duplicate retention labels are created.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/bulk-create-publish-labels-using-powershell.md]:
```suggestion
- If the .csv file contains a retention label with the same name as one that already exists, the script skips creation of that retention label. No duplicate retention labels are created.
```

File: SecurityCompliance/bulk-create-publish-labels-using-powershell.md [hunk 5]
```diff
@@ -708,9 +708,9 @@ Follow the steps here:
   
 - [Connect to Office 365 Security &amp; Compliance Center PowerShell](https://go.microsoft.com/fwlink/?linkid=799771)
     
-## Step 5: Run the PowerShell script to create and publish the labels
+## Step 5: Run the PowerShell script to create and publish the retention labels
 
-After you've connected to Security &amp; Compliance Center PowerShell, next you run the script that creates and publishes the labels.
+After you've connected to Security &amp; Compliance Center PowerShell, next you run the script that creates and publishes the retention labels.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/bulk-create-publish-labels-using-powershell.md]:
```suggestion
Connect to Security &amp; Compliance Center PowerShell and run the script that creates and publishes the retention labels.
```

File: SecurityCompliance/bulk-create-publish-labels-using-powershell.md [hunk 6]
```diff
@@ -728,7 +728,7 @@ After you've connected to Security &amp; Compliance Center PowerShell, next you
 
 ## Step 6: View the log file with the results
 
-When you run the script, it generates a log file that records each action it took and whether the action succeeded or failed. The log file includes all metadata about what labels were created and what labels were published. You can find the log file at this location -- note that the digits in the file name vary.
+When you run the script, it generates a log file that records each action it took and whether the action succeeded or failed. The log file includes all metadata about the retention labels created and the retention labels published. You can find the log file at this location -- note that the digits in the file name vary.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/bulk-create-publish-labels-using-powershell.md]:
```suggestion
When you run the script, it generates a log file that records each action it took and whether the action succeeded or failed. The log file includes all metadata about the retention labels created and the retention labels published. You can find the log file at this location (note that the digits in the file name vary):
```

**Pull request merged**